x/tokenfactory/types/expected_keepers.go

```diff
@@ -20,10 +20,14 @@ type BankKeeper interface {
 
 	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
 	HasBalance(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coin) bool
+	GetAllBalances(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
```

Comment:
Are these expected keeper changes only needed for test code? Maybe another one that extends them, so it is clear production code requires X and tests X + Y

Reply:
Yeah, these expected func in bank keeper only need for sims test code